src/olympia/versions/admin.py

```diff
@@ -17,14 +17,15 @@ class Media:
         js = ('js/admin/l10n.js',)
 
     view_on_site = False
-    readonly_fields = ('version', 'channel', 'recommendation_approved')
+    readonly_fields = ('id', 'created', 'version', 'created', 'channel',
```

Comment:
Do you need `created` twice?

Reply:
Oops, no. Copy & paste mistake.

Reply:
Fixed in https://github.com/mozilla/addons-server/pull/14189/commits/b46b302234dfd06ddade978c0867c4b937e08f5a.